<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TGroup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 36, 36);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>321</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>341</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"> QTableWidget {
        color: #ffffff;
        border-radius: 10px;
    }

QHeaderView::section {
    background-color: rgb(39, 39, 39);
	color: #ffffff;
    border: none;
    height: 28px;
}

QTableCornerButton::section {
	background-color: rgb(66, 66, 66);
}

QTableWidget::item {
	border: 0px; 
	padding: 5px; 
	color: #ffffff;
    margin-top: 5px;          
    border-radius: 9px;
    padding-left: 5px;
}

/* VERTICAL SCROLLBAR */
 QScrollBar:vertical {
	border: none;
    background: transparent;
    width: 8px;
    margin: 15px 0 15px 2px;
	border-radius: 0px;
    margin-right: 4px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(158, 158, 158);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(66, 66, 66);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}



/* HORIZONTAL SCROLLBAR - HOMEWORK */
QScrollBar:horizontal {
   	border: none;
	background-color: transparent;
	height: 3px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::handle:horizontal {
    	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::add-line:horizontal {
    
}
QScrollBar::sub-line:horizontal {
    
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{

}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>95</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Groups</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Edit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delete</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-style: solid;
font: 900 9pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border-bottom-width: 1px;
border-color: none none white none;
border-radius:0px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 7pt &quot;Arial Black&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Group Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchGameExeBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#searchGameExeBtn { 
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
        border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton#searchGameExeBtn:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton#searchGameExeBtn::focus:pressed{ 
    background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>+ Add new app</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: transparent;
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton::focus:pressed{ 
	background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"> QTableWidget {
        color: #ffffff;
        border-radius: 10px;
    }

QHeaderView::section {
    background-color: rgb(39, 39, 39);
	color: #ffffff;
    border: none;
    height: 28px;
}

QTableCornerButton::section {
	background-color: rgb(66, 66, 66);
}

QTableWidget::item {
	border: 0px; 
	padding: 5px; 
	color: #ffffff;
    margin-top: 5px;          
    border-radius: 9px;
    padding-left: 5px;
}

/* VERTICAL SCROLLBAR */
 QScrollBar:vertical {
	border: none;
    background: transparent;
    width: 8px;
    margin: 15px 0 15px 2px;
	border-radius: 0px;
    margin-right: 4px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(158, 158, 158);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(66, 66, 66);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}



/* HORIZONTAL SCROLLBAR - HOMEWORK */
QScrollBar:horizontal {
   	border: none;
	background-color: transparent;
	height: 3px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::handle:horizontal {
    	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::add-line:horizontal {
    
}
QScrollBar::sub-line:horizontal {
    
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{

}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>App</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delete</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="searchGroupIcon">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#searchGroupIcon { 
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
        border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton#searchGroupIcon:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton#searchGroupIcon::focus:pressed{ 
    background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>Group Image</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"> QTableWidget {
        color: #ffffff;
        border-radius: 10px;
    }

QHeaderView::section {
    background-color: rgb(39, 39, 39);
	color: #ffffff;
    border: none;
    height: 28px;
}

QTableCornerButton::section {
	background-color: rgb(66, 66, 66);
}

QTableWidget::item {
	border: 0px; 
	padding: 5px; 
	color: #ffffff;
    margin-top: 5px;          
    border-radius: 9px;
    padding-left: 5px;
}

/* VERTICAL SCROLLBAR */
 QScrollBar:vertical {
	border: none;
    background: transparent;
    width: 8px;
    margin: 15px 0 15px 2px;
	border-radius: 0px;
    margin-right: 4px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(158, 158, 158);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(66, 66, 66);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: transparent;
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}



/* HORIZONTAL SCROLLBAR - HOMEWORK */
QScrollBar:horizontal {
   	border: none;
	background-color: transparent;
	height: 3px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::handle:horizontal {
    	background-color: rgb(158, 158, 158);
	min-height: 30px;
	border-radius: 1px;
}
QScrollBar::add-line:horizontal {
    
}
QScrollBar::sub-line:horizontal {
    
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{

}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>App</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delete</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 7pt &quot;Arial Black&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Group Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-style: solid;
font: 900 9pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border-bottom-width: 1px;
border-color: none none white none;
border-radius:0px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchGameExeBtn_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
        border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton::focus:pressed{ 
    background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>+ Add new app</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="updateBtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: transparent;
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton::focus:pressed{ 
	background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchGroupIcon_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#searchGroupIcon_2 { 
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
        border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton#searchGroupIcon_2:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton#searchGroupIcon_2::focus:pressed{ 
    background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
      </property>
      <property name="text">
       <string>Group Image</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="newGroupBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: transparent;
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton::focus:pressed{ 
	background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
    </property>
    <property name="text">
     <string>New Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groupsBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: transparent;
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 1px;
	border-color: rgb(255, 255, 255);
	border-style: outset;
border-radius: 10px;
}

QPushButton:hover{ 
    background-color: rgb(252, 196, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}

QPushButton::focus:pressed{ 
	background-color: rgb(252, 72, 25);
	font: 900 9pt &quot;Arial Black&quot;;
	color: rgb(255, 255, 255);
    border: 0px;
}
</string>
    </property>
    <property name="text">
     <string>Groups</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
